README.md

```diff
@@ -25,12 +25,11 @@ Running the Back-end & Front-end
     
 Once Docker and Python are Installed then do the following:
     
-    copy server/src/configs/sample_config.py -> server/src/configs/config.py
+    cp server/sample-bluebutton-config.py server/.bluebutton-config.py
```

Comment:
I think you meant .json instead of .py here.

Reply:
@JFU-GIT @ajshred   I ran in to the same issue with the config file.

I pushed up a README.md update and also added some info for the SDK usage.

I think the README.md needs some more editing/finishing. Would be good to have a documentation review Jira story, if there isn't one in the backlog.

Reply:
Cool thanks for the inputs